azure/packages/azure-client/package.json

```diff
@@ -30,12 +30,15 @@
 		"api": "fluid-build . --task api",
 		"api-extractor:commonjs": "api-extractor run --local",
 		"api-extractor:esnext": "copyfiles -u 1 \"dist/**/*-@(alpha|beta|public|untrimmed).d.ts\" lib",
+		"api-extractor:rename-types": "renamer \"lib/**\" -f .d.ts -r .d.mts --force",
+		"api-extractor:rewrite-type-imports": "replace-in-file \"/from \\\"\\\\.((.*?)(\\\\.mjs)?)\\\";/g\" \"from \\\".$2.mjs\\\";\" \"lib/**/*.d.mts\" --isRegex",
```

Comment:
I wish this didn't take so much escaping, but it's easier than writing and shipping a tool to do it.